### PR TITLE
Add and remove event attendees

### PR DESCRIPTION
**Pull request by NjaalSoerland**
Some styling that needs to be done, but it makes more sense to fix that when dealing with attendee pools. So this should be merged first

### CODE REVIEW

File: apps/gateway-email/package.json
Comment (NjaalSoerland) [inline apps/gateway-email/package.json:1]: necessary to be able to run pnpm dev. Didn't mean to push but don't know how to make it work locally with this script

File: apps/gateway-email/package.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "docker run -p 9000:8080 -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_REGION=eu-north-1 gateway-email:latest",
```
Comment (NjaalSoerland) [inline apps/gateway-email/package.json:7]: necessary to be able to run pnpm dev. Didn't mean to push but don't know how to make it work locally with this script

Comment (junlarsen) [reply apps/gateway-email/package.json:7]: Can you rename this and dispatcher-auth's dev scripts to "docker:dev"?

**Review by junlarsen — CHANGES_REQUESTED** [review]
LGTM, just change the script names and rebase on main first

File: apps/dashboard/src/components/GenericSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { Autocomplete } from "@mantine/core"
+
+interface GenericSearchProps<T> {
+  onSearch: (query: string) => void
+  onSubmit: (item: T) => void
+  items: T[]
+  dataMapper: (item: T) => string
+}
+
+const GenericSearch = <T,>({ onSearch, onSubmit, items, dataMapper }: GenericSearchProps<T>) => {
```
Comment (junlarsen) [inline apps/dashboard/src/components/GenericSearch.tsx:11]: What if we make GenericSearchProps also include the mantine Autocomplete props so you can override placeholder if you want to?

**Review by junlarsen — APPROVED** [review]
LGTM with a suggestion maybe for anoter PR

**Pull request merged**